domain/src/main/java/com/example/domain/movies/observers/ObserveMovieDetails.kt

```diff
@@ -0,0 +1,22 @@
+package com.example.domain.movies.observers
+
+import com.example.core.data.movies.MoviesCatchSource
+import com.example.domain.SubjectInteractor
+import com.example.model.Movie
+import data.movies.MoviesRepository
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.map
+import javax.inject.Inject
+
+class ObserveMovieDetails @Inject constructor(
```

Comment:
lets refactor this interactor in way it should not get the store type param
find a way to flat all the movies from all stores and return the movie with id

Reply:
ok